Composer/packages/server/src/controllers/publisher.ts

```diff
@@ -52,7 +52,7 @@ export const PublishController = {
     // append config from client(like sensitive settings)
     const configuration = {
       name: profile.name,
-      ...JSON.parse(profile.configuration),
+      customizeConfiguration: JSON.parse(profile.configuration),
```

Comment:
this will be confusing to developers who will expect the configuration to just be in the main object. why create this sub-object?

Reply:
Because it's easy to have duplicated name between user-inputed configuration and name(profileName). Or we changed the name into profileName?